src/google/protobuf/compiler/cpp/cpp_file.cc

```diff
@@ -824,19 +824,11 @@ void FileGenerator::GenerateNamespaceClosers(io::Printer* printer) {
 
 void FileGenerator::GenerateForwardDeclarations(io::Printer* printer) {
```

Comment:
Where is this GenerateForwardDeclarations() method called? Can you remove that callsite instead?